packages/platforms/accton/x86-64/amx3200/Makefile

```diff
@@ -0,0 +1 @@
+include $(ONL)/make/pkg.mk
```

Comment:
Please use UNIX style line endings.

Reply:
We follow this file of  other platform, like as9926 
What is the "UNIX style line endings" you say?